src/sketchup_objects/link.rb

```diff
@@ -265,10 +264,8 @@ def recreate_children
   def create_children
     create_elongations
 
-    link_position = @position.offset(@first_elongation.direction)
-
     add(@first_elongation,
-        BottleLink.new(link_position, direction, @model, id: @id),
+        BottleLink.new(@position, direction, @model, id: @id),
```

Comment:
Did you check how these changes look like with the conventional bottle models?

Reply:
This makes it look a little bit weird, cause the bottles are scaled, but it does not break anything.